src/bin/gpfdist/gpfdist.c

```diff
@@ -533,6 +535,7 @@ static void usage_error(const char* msg, int print_usage)
 #endif
 #ifdef GPFXDIST
 					    "        -c file    : configuration file for transformations\n"
+					    "        -X name    : name of default transformation\n"
```

Comment:
`gpfdist --help` takes help from this file in case `$GPHOME` is available.
https://github.com/arenadata/gpdb/blob/4003b8f0fc15476e7bb19b667cebac269f8a7867/src/bin/gpfdist/gpfdist.c#L495-L501
It's source is located in `gpMgmt/doc/gpfdist_help`. I think it should be updated too.

Reply:
True. Fixed.

Reply:
Flags are shown in the usage too:
https://github.com/arenadata/gpdb/blob/0dc96e8dca4c27aacde659f7f7f7bcb2516e7fc2/gpMgmt/doc/gpfdist_help#L10-L12
How about adding `[-X <default transformation>]` there too?

Reply:
Done